code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -55,6 +55,13 @@
 	/// true whenever someone with the strong pull component is dragging this, preventing opening
 	var/strong_grab = FALSE
 
+	var/obj/effect/overlay/closet_door/door_obj
+	var/is_animating_door = FALSE
+	var/door_anim_squish = 0.12
+	var/door_anim_angle = 136
+	var/door_hinge_x = -6.5
+	var/door_anim_time = 2.0 // set to 0 to make the door not animate at all
```

Comment:
cool magic numbers.

Reply:
Do you need the units (seconds, degrees, coordinates) explicitly labelled? The values exist because different types of closets (specifically, freezers) have different angles for the door to swing open.

Reply:
what I meant to point out was door_anim_squish, the others are fine.